proxy-robots-module/src/main/java/orwell/proxy/robot/RobotInputSetVisitor.java

```diff
@@ -0,0 +1,79 @@
+package orwell.proxy.robot;
+
+import com.google.protobuf.InvalidProtocolBufferException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import orwell.messages.Controller;
+
+
+/**
+ * Created by Michaël Ludmann on 5/12/15.
+ */
+public class RobotInputSetVisitor implements IRobotInputVisitor {
```

Comment:
This visitor visits all the actuators (for now: motors for Move, and speaker for Fire) of the Robot that are able to receive and act on an Input.
I've used the Input name to remain consistent with the Protobuf message.

I'm having a hard time finding another name for it. Open to suggestions.